### PR TITLE
Add README to `/framework`

### PR DESCRIPTION
**Pull request by idea404**
This PR adds a simple README to `@near-lake/framework`.

### CODE REVIEW

File: packages/near-lake-framework/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+```ts
+import { startStream, types } from '@near-lake/framework';
+
+const lakeConfig: types.LakeConfig = {
+    s3BucketName: 'near-lake-data-mainnet',
+    s3RegionName: 'eu-central-1',
+    startBlockHeight: 66264389,
+};
+
+async function handleStreamerMessage(
```
Comment (roshaans) [inline packages/near-lake-framework/README.md:18]: @khorolets perhaps we should change this function handler name to `handleBlock`

File: packages/near-lake-framework/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    block: types.Block,
+    context: types.LakeContext
+): Promise<void> {
+    let events = block.eventsByAccountId("x.paras.near")
```
Comment (roshaans) [inline packages/near-lake-framework/README.md:22]: Adding a comment within the function will make it a bit more clear that this is the main place where the user of the library may add any code. 


```suggestion
    // custom logic for handling the block
    let events = block.eventsByAccountId("x.paras.near")
```

File: packages/near-lake-framework/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    console.log(events)
+}
+
+(async () => {
+    await startStream(lakeConfig, handleStreamerMessage);
+})();
+```
+
+The interface to capture data about an event has the following arguments:
```
Comment (roshaans) [inline packages/near-lake-framework/README.md:31]: I think the following may be unnecessary information for an indexer developer to know. This should be presented in the ReadMe for @near-lake/primitives

File: packages/near-lake-framework/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+): Promise<void> {
+    let events = block.eventsByAccountId("x.paras.near")
+    console.log(events)
+}
+
+(async () => {
+    await startStream(lakeConfig, handleStreamerMessage);
+})();
+```
+
```
Comment (roshaans) [inline packages/near-lake-framework/README.md:30]: Some more concrete words to explain the example and where to find documentation may be helpful.


```suggestion
The main function in this example is handleStreamerMessage(block, context), which contains custom logic for how to handle a given block. `block` is of type Block from @near-lake/primitives which gives access to helper methods to make it easier to extract data. You may find a list of helper functions [here](https://www.npmjs.com/package/@near-lake/primitives). 
```

**Review by roshaans — COMMENTED** [review]
Once published, we can add better links to documentation, but this is a good start! Added some minor additions that I believe make things a bit more clearer.

File: packages/near-lake-framework/README.md (hunk 3 above)
Comment (idea404) [reply packages/near-lake-framework/README.md:32]: Fine by me, happy to make the change later on today or tomorrow unless anybody else thinks otherwise

**Review by khorolets — APPROVED** [review]
A few minor changes and we're good to merge. I am approving to prevent this PR from being blocked on me

**Pull request merged**